MAINTENANCE.md

```diff
@@ -208,3 +209,75 @@ Here are some of the shortcuts included in [`bashrc_for_gmt`](/admin/bashrc_for_
   opening each file.
+
+## Continuous Integration
+
+We use GitHub Actions Continuous Integration (CI) services to build and test the project on Linux, macOS and Windows.
+
+There are 11 configuration files located in `.github/workflows/`:
+
+1. `backport.yml` (Backports PRs into the 6.1 branch)
+
+  This workflow backports PRs labels by "backport 6.1" into the 6.1 branch.
+
+2. `build.yml` (Build GMT and run a few simple tests)
+
+  This workflow is run on every commit to the *master* branch and Pull Request branches. The workflow configures
+  and builds GMT on Linux, macOS, and Windows and runs some simple tests. The workflow uses the scripts in the `ci/`
+  directory for downloading/installing dependencies (`ci/download-coastlines.sh` and `ci/install-dependencies-*.sh`),
+  configuring GMT (`ci/config-gmt-*.sh`), building GMT (`ci/build-gmt.sh`), and running some simple
+  tests (`ci/simple-gmt-tests.*`).
+
+3. `check-links.yml` (Check links in the repository and website)
+
+  This workflow is run every Sunday at 12:00 (UTC) to check all external links in plaintext and HTML files. It will
+  create an issue if broken links are found.
+
+4. `ci-caches.yml` (Cache GMT remote data files and Windows vcpkg libraries needed for GitHub Actions CI)
+
+  This workflow is run every Sunday at 12:00 (UTC). If new remote files are needed urgently, maintainers can
+  manually uncomment the 'pull_request:' line in the ``ci-caches.yml`` file to refresh the cache. The workflow uses the
+  script `ci/install-dependencies-windows.sh` to cache the vcpkg libraries.
+
+5. `code-validator.yml` (Validate code consistency)
+
+  This workflow is run on every commit to the *master* branch and Pull Request branches. It runs the scripts
+  `src/gmt_make_PSL_strings.sh`, `src/gmt_make_enum_dicts.sh`, and `src/gmt_make_module_purpose.sh` and exits with an
+  error if any changes are detected by git. It also checks the GMT version year in `ConfigDefault.cmake` and the
+  permissions of the bash scripts.
+
+6. `docker.yml` (Build GMT on different Linux distros using dockers)
+
+  This workflow is run when Pull Requests are merged into the *master* branch. It downloads/installs dependencies
+  and configures and builds GMT on different Linux distributions using dockers.
+
+7. `docs.yml`  (Build documentation on Linux/macOS/Windows)
+
+  This workflow is run when Pull Requests are merged into the *master* branch, if the Pull Request involved changes to
+  files in the `doc/`, `.github/workflows/`, or `ci/` directories.
+
+  The workflow also handles the documentation deployment:
+
+   * Updating the development documentation by pushing the built HTML pages from the *master* branch into the `dev`
+     folder of the *gh-pages* branch.
+   * Updating the `latest` documentation link to the new release.
+
+8. `draft-release.yml` (Drafts the next release notes)
+
+  This workflow is run to update the next releases notes as Pull Requests are merged into the *master* branch.
```

Comment:
@seisman, I am not sure I understood this workflow. Could you please update if needed?